zerver/tests/test_events.py

```diff
@@ -1061,6 +1061,37 @@ def test_invitation_accept_invite_event(self) -> None:
         error = schema_checker('events[4]', events[4])
         self.assert_on_error(error)
 
+    def test_send_more_than_one_invite_to_same_user(self) -> None:
```

Comment:
This doesn't test events, so it shouldn't be in this file. test_signup.py is more suitable